Pipfile

```diff
@@ -11,18 +11,14 @@ python_version = "3.6.3"
 
 flask = "==0.12.2"
 flask-cors = "==3.0.3"
-gunicorn = "==19.7.1"
-itsdangerous = "==0.24"
 jsonschema = "==2.6.0"
```

Comment:
Do we need jsonschema?

Reply:
There are a few imports in here which aren't used currently but will be. I suppose I shouldn't have put these in already.